tests/local/network.go

```diff
@@ -90,7 +90,7 @@ func NewLocalNetwork(
 	Expect(err).Should(BeNil())
 	warpChainConfigPath := f.Name()
 
-	allNodes := slices.Clone(extraNodes) // to be appended w/ subnet validators
+	allNodes := extraNodes // to be appended w/ subnet validators
```

Comment:
the linter made me do it! :) it gave me some seemingly bogus type checking error that it couldn't infer the output type of `slices.Clone`, so I just did it a different way. I'm not 100% sure why the linter didn't catch this earlier.

later on in this code, we want to iterate over `allNodes`, which includes not just `extraNodes` but also all of the subnet nodes as well. this declaration is me starting to build up the list off all nodes, starting with the `extraNodes`.

Reply:
Ah yeah, I understand that practically it's not necessary here just a best practice to not have two slice variables pointing to the same underlying arrays. FWIW I think it would have been solved by declaring `var allNodes []*tmpnet.Node` beforehand.